src/main/scala/com/fulcrumgenomics/commons/CommonsDef.scala

```diff
@@ -25,15 +25,13 @@
 package com.fulcrumgenomics.commons
 
 import java.io.Closeable
-import java.util.Comparator
+import java.util.concurrent.ForkJoinPool
```

Comment:
Imports were optimized (with IntelliJ) and one deprecation warning was solved with this import:

Deprecated import (alias):

https://www.scala-lang.org/api/current/scala/concurrent/forkjoin/package$$ForkJoinPool$.html

Reply:
Unfortunately we can't do this while we're still cross-building for 2.11 and 2.12 - it breaks the 2.11 build.